### ISSUE TITLE
Alert-Modul Error after automatic hide.

### ISSUE THREAD

**Issue by desq42** [MichMich/MagicMirror#283]
_"notificationFx.js:147 Uncaught NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.onEndAnimationFn @ notificationFx.js:147_

Could we have a "ns-custom.css"? I´m having issues showing my motioneye snapshot.
![bildschirmfoto 2016-05-03 um 15 33 12](https://cloud.githubusercontent.com/assets/18470786/14984833/02b3ea08-1145-11e6-9400-d0f563516325.png)
Using:
```javascript
this.sendNotification("SHOW_ALERT", {title: "Motion", imageHeight: "300px", timer: "6000", imageUrl: "http://localhost:8080/modules/cam/cam.png"}); 
```

**Comment by MichMich** [comment]
There is a css/custom.css file which might help. For for the other issues: @paviro :)

**Comment by desq42** [comment]
I already tried that. My css/custom.css-file is overridden by "ns-default.css".
![overridden](https://cloud.githubusercontent.com/assets/18470786/14986391/f83a2bb2-114b-11e6-81a9-c48d8df58880.png)

**Comment by MichMich** [comment]
Ah, that makes sense ... This is something I need to change. The module css files are currently loaded after the custom.css file. Could you make a separate issue for this? We can then keep this issue for the bug mentioned in the openings post.

**Comment by desq42** [comment]
done. New seperate issue #284

**Comment by MichMich** [comment]
This issue should be fixed. Can you check it?

**Comment by desq42** [comment]
fixed with: #284;

**Comment by paviro** [comment]
Really? Wanted to check today, well glad it works again :D
